src/lib/bootstrap.php

```diff
@@ -19,6 +19,19 @@
 $loader = require __DIR__.'/../app/autoload.php';
 ZLoader::register($loader);
 
+// set current domain for using it as container parameter
+// http://symfony.com/doc/current/cookbook/configuration/external_parameters.html
+if (strpos(__DIR__, '/src/lib') !== false) {
+    $_SERVER['SYMFONY__ASSETS_BASE_URL'] = '//' . $_SERVER['HTTP_HOST'] . '/src/web';
+} else {
+    $_SERVER['SYMFONY__ASSETS_BASE_URL'] = '//' . $_SERVER['HTTP_HOST'] . '/web';
+}
+
```

Comment:
`%assets_base_url%` is an own (a custom) parameter which has no default value. This is why composer fails when no value is set in the config. But as soon as I set a (default) value in parameters.yml the dynamic value from the bootstrap code is not set/used.

I will now check if it behaves differently if we remove the config parts and directly set the original value in bootstrap instead of introducing a custom parameter at all.
```
$core->getContainer()->setParameter('framework.templating.assets_base_urls', $assetsBaseUrl);
```